esque/controller/topic_controller.py

```diff
@@ -180,10 +180,12 @@ def _get_partition_data(
         for partition_id, meta in confluent_topic.partitions.items():
             low = int(low_watermarks[partition_id].offset[0])
             high = int(high_watermarks[partition_id].offset[0])
+            latest_timestamp = None
             if high > low:
-                latest_timestamp = float(consumer.consume(high - 1, partition_id).timestamp()[1]) / 1000
-            else:
-                latest_timestamp = None
+                try:
+                    latest_timestamp = float(consumer.consume(high - 1, partition_id).timestamp()[1]) / 1000
+                except MessageEmptyException:
+                    """Due to timeout latest timestamp is missing."""
```

Comment:
Can you use `log.warning` for that? And include both topic name and partition id?
You'll need to put
```python
import logging
log = logging.getLogger(__name__)
```
At the beginning of the file to get the logger.